posthog/api/test/test_trends.py

```diff
@@ -112,30 +116,106 @@ def test_includes_only_intervals_within_range(client: Client):
         }
 
 
+@pytest.mark.django_db
+@pytest.mark.ee
+def test_math_is_functional(client: Client):
+    """
+    Regression test for https://github.com/PostHog/posthog/issues/4858
+
+    The expectation of the frontend for certain values of `display` (here it is
+    "ActionsTable") is to return a single aggregate value for each "action"
+    aggregation requested. i.e. get me the number of distinct users who
+    performed action x, over an arbitrary time range. This was all working fine,
+    however, the frontend also performs some calculations, for example the mean
+    of the number of distinct users by day. It was expecting a daily interval to
+    do this, which we were not returning. We now always produce both the total
+    value, and the interval breakdowns.
+
+    NOTE: this was functioning as expected for postgresql as an events backend.
+    """
+    signup_ok(client=client, email="user@fake.fake", password="password", organization_name="Hedgehogs United, LLC")
+    project = get_current_project_ok(client=client)
+
+    # "2021-09-19" is a sunday, i.e. beginning of week
+    with freeze_time("2021-09-20T16:00:00"):
+        #  First identify as a member of the cohort
+        distinct_id = "abc"
+        identify(distinct_id=distinct_id, team_id=project["id"])
+
+        capture_event(
```

Comment:
:-1: Not a fan of this pattern at all.

Reply:
Cases as if I was the plugin-server / (event ingest):

 1. Append Event  - create_event()
 2. Create Person + PDI + Append Event - create_person()
 3. Update Person (append new row + group by query)  + Append Event
 4. Create PDI + Append Event
 5. Merge Person (update in postgres, Append new row + group by query) + new PDI

Only need 1 + 2 for testing queries / posthog. Cohorts/actions also required.

@hazzadous to update to use `create_event()` and `create_person()`